include/crm/pengine/pe_types_compat.h

```diff
@@ -230,6 +230,7 @@ typedef struct pe_action_wrapper_s pe_action_wrapper_t;
 //! \deprecated Use pcmk_node_t instead
 typedef struct pe_node_s node_t;
 
+// @COMPAT This is used by sbd (as of at least 1.5.2)
```

Comment:
I don't see this or `pe_resource_t` in sbd v1.5.2 code

Reply:
Ah these comments were left over from earlier work. Nope they are not used

Reply:
Dropped comments and added commits to delete those two types